### ISSUE TITLE
[FEATURE] - Option for enable location function

### ISSUE THREAD

**Issue by wuyilingwei** [Atlas-OS/Atlas#1127]
### Before continuing...

- [X] I agree to follow Atlas' [Code of Conduct](https://github.com/Atlas-OS/.github/blob/main/profile/CODE_OF_CONDUCT.md)
- [X] I have searched our [issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to verify that the issue hasn't been reported already
- [X] I have searched the [Atlas documentation](https://docs.atlasos.net) beforehand for a solution
- [X] I am on the latest version of Atlas, and didn't do an unsupported upgrade

### What is your feature request regarding to?

Atlas Playbook

### Is your feature request related to a problem? Please describe.

I cannot enable location in win10/11, but some of my devices need it (like surface).

### Describe the solution you would like.

We can choice enable or disable location in atlas setting.

### Describe alternatives you have considered.

I reset reg HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\CapabilityAccessManager\ConsentStore\location back to Allow and reboot, but it still cannot enable location (no option) in setting.

**Comment by he3als** [comment]
As a workaround, run these commands in Command Prompt and restart:
```
sc config lfsvc start=demand
sc config MapsBroker start=auto
"%windir%\AtlasModules\Scripts\settingsPages.cmd" /unhide privacy-location
```
This is planned for the next release.

**Comment by wuyilingwei** [comment]
> 解决方法是在命令提示符下运行以下命令，然后重新启动：
> 
> ```
> sc config lfsvc start=demand
> sc config MapsBroker start=auto
> "%windir%\AtlasModules\Scripts\settingsPages.cmd" /unhide privacy-location
> ```
> 
> 计划在下一个版本中提供一个选项。



> As a workaround, run these commands in Command Prompt and restart:
> 
> ```
> sc config lfsvc start=demand
> sc config MapsBroker start=auto
> "%windir%\AtlasModules\Scripts\settingsPages.cmd" /unhide privacy-location
> ```
> 
> An option is planned for the next release.

Thanks for your help. @.@

**Comment by he3als** [comment]
Fixed in [release v0.4.1](https://github.com/Atlas-OS/Atlas/releases/tag/0.4.1). See the release for more info on what v0.4 users can do.
